package.json

```diff
@@ -51,7 +51,7 @@
     ]
   },
   "dependencies": {
-    "codemirror": "^5.26.0",
+    "codemirror": "^5.46.0",
```

Comment:
is the `codemirror` upgrade a requirement? we will need a lockfile change, or this will fail the `yarn check --integrity` we added to  CI yesterday

Reply:
@acao I've updated the lock file